packages/docs/docs/guides/5-request-client/4-btc-payment-detection.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * # Creating a Request using the Bitcoin payment network
+ *
+ * This is an example of creating a request using the Bitcoin payment network
+ * The request will be stored in memory and cleared as soon as the script is finished running.
+ */
+
+/**
+ * ## Basics
+ * 
+ * Before creating a request, the Request Client must be initialized.
+ */
+
+/**
+ * ### Imports
+ *
+ * First we import the 2 packages we will need to create the request:
+ */
+// The signature provider allows us to sign the request
+import { EthereumPrivateKeySignatureProvider } from '@requestnetwork/epk-signature';
+// RequestNetwork is the interface we will use to interact with the Request network
+import * as RequestNetwork from '@requestnetwork/request-client.js';
+
+/**
+ * ### Identity
+ *
+ * To create a request we need to declare the identities of the parties involved.
+ * Identities are the unique identifier of a request user. They are generally different from payment addresses, but can be the same. They identify an entity like a person or business.
+ */
+
+// Here we declare the payee identity, with the payee identity ethereum address
+const payeeIdentity = {
+  type: RequestNetwork.Types.Identity.TYPE.ETHEREUM_ADDRESS,
+  value: '0x627306090abab3a6e1400e9345bc60c78a8bef57',
+};
+
+// Here we declare the (optional, but recommended) payer identity address.
+const payerIdentity = {
+  type: RequestNetwork.Types.Identity.TYPE.ETHEREUM_ADDRESS,
+  value: '0xF317BedAA5c389F2C6f469FcF25e0752C7228Ba6',
+};
+
+/**
+ * ### Signature Provider
+ *
+ * The Ethereum private key signature provider allows a user to pass in their private Ethereum key to sign a request. The signer is either the payee or the payer.
+ * The signature is a proof of who created the request and of it's integrity (that no data changed after it was signed).
+ * This process is similar to the signature of an Ethereum transaction.
+ */
+
+// The signature info requires the request creator private key.
+// For this demo purposes, we hard-coded the private key. Please be careful with how you store and handle your private key since it's a very sensitive piece of data.
```

Comment:
Shouldn't we advise on specific solutions or designs to workaround this issue?

Reply:
It would be good to have more details about the signature somewhere in the documentation. But I guess adding it here would make the document less focused on it's intent (BTC payment).

I expect developers dealing with request to have a basic knowledge on security and how to handle passwords and private keys. Maybe this assumption is wrong...

We should have a section with: `Using the metamask signature provider` and `Writing your own signature provider`, for example.

Reply:
I agree we should have resources in the docs for signature management. I don't think it's relevant here though since it's focused on the payment network
I JIRAed the issue: https://requestnetwork.atlassian.net/browse/PROT-1167